.storybook/preview-head.html

```diff
@@ -3,30 +3,30 @@
         background: white;
     }
     
-    @font-Face{font-family:"Guardian Text Egyptian Web"; font-weight:400; src:url("./public/fonts/GuardianTextEgyptian-Reg.ttf");}
```

Comment:
I suspect the dot here was unintentional as it would have resolved a different font depending on how many slashes there are in the URL?